userspace/libscap/scap_vtable.h

```diff
@@ -31,9 +31,8 @@ typedef struct scap scap_t;
 typedef struct ppm_evt_hdr scap_evt;
 
 enum scap_eventmask_op {
-	SCAP_PPM_SC_MASK_ZERO = 0, //< disable all syscalls
-	SCAP_PPM_SC_MASK_SET = 1, //< enable a syscall
-	SCAP_PPM_SC_MASK_UNSET = 2, //< disable a syscall
+	SCAP_PPM_SC_MASK_SET = 0, //< enable a syscall
+	SCAP_PPM_SC_MASK_UNSET = 1, //< disable a syscall
```

Comment:
Is there any combination of kernel/userspace components where renumbering these constants will break? I hope not but I'd feel minimally better about it if we kept set=1 and unset=2 (leaving 0 unused) or just renumbered these somewhere completely different (e.g. set=3 unset=4).

Reply:
There should not be any concern here since i also bumped driver maj API_VERSION.
We can leave them, i just didn't like it starting from 1 instead of 0 (and the fact that it is different from `scap_tp_mask_op` below).
But i will reset them to their values, and just comment the first `SCAP_PPM_SC_MASK_ZERO line (like i did for kmod ioctls)

Reply:
Force-pushed!